src/internal/rest.rs

```diff
@@ -1,21 +1,17 @@
 //! Helpers for talking to the ironcore service.
 
-use chrono::{DateTime, Utc};
-use futures::{stream::Stream, Future, IntoFuture};
-use reqwest::{
-    header::HeaderMap,
-    r#async::{Chunk, Client as RClient, Request as ARequest},
-    Method, StatusCode, Url,
-};
-use serde::{de::DeserializeOwned, Serialize};
-use std::marker::PhantomData;
-
 use crate::internal::{
     auth_v2::AuthV2Builder, user_api::UserId, DeviceSigningKeyPair, IronOxideErr, Jwt,
     RequestErrorCode, OUR_REQUEST,
 };
-use futures::future::Either;
+use bytes::Bytes;
+use chrono::{DateTime, Utc};
 use percent_encoding::SIMPLE_ENCODE_SET;
+use reqwest::{
+    header::HeaderMap,
+    r#async::{Client as RClient, Request as ARequest},
```

Comment:
```suggestion
    r#async::{Client as RClient, Request},
```

Can this just be `Request` now, or something more specific like `Reqwest` if the intent is to differentiate it from `IronCoreRequest`?

Reply:
@clintfred ^

Reply:
I agree that we should just leave it as `Request`. `ARequest` actually came in when we were also using the blocking (then default) `Request` as well.

Reply:
Fixed.